qabelbox/src/main/java/de/qabel/qabelbox/fragments/CreateIdentityEditTextFragment.java

```diff
@@ -30,6 +32,16 @@ public static CreateIdentityEditTextFragment newInstance(int messageId, int edit
         return fragment;
     }
 
+    public static CreateIdentityEditTextFragment newInstance(int messageId, int editTextHintId, BaseWizardActivity.NextChecker checker, int textInputType) {
+
+        CreateIdentityEditTextFragment fragment = new CreateIdentityEditTextFragment();
+        fragment.mMessageId = messageId;
+        fragment.mEditTextHintId = editTextHintId;
+        fragment.mChecker = checker;
+        fragment.inputType = textInputType;
+        return fragment;
```

Comment:
```JAVA
CreateIdentityEditTextFragment fragment = newInstance(messageId, editTextHintId, checker);
fragment.inputType = textInputType;
return fragment;
```
by reusing the other newInstance method the fact stands out that only `fragment.inputType = textInputType;` was added and no functionality that is only added to the other newInstance method will be lost. (and it's more DRY)